docs/src/Experimental/galois.md

```diff
@@ -0,0 +1,207 @@
+
+ - the *automorphism group*
+ - the *Galois group*
+ 
+The automorphism group deals with the actual automorphism of `K` fixing `k`
+and thus is, in general trivial. Access to via two constructions:
+
+ - a list of all automorphisms (usually only the identity)
+ - the group of automorphisms, returned as an abstact group and
+   a map linking group elements to actual automorphisms
+
+On the other hand, the Galois group is isomorphic to the automorphism
+group of the normal closure and is explicitly given as a group of
+permutations of the roots  of the defining polynomial. Thus even
+in the case of `K` over `k` being normal, elements of the
+Galois group do not immediately give automorphisms at all.
+
+Currently, the computation of Galois groups is possible for
+ 
+  - `K` a simple extension of the rationals (`AnticNumberField`)
+  - `K` a simple extension of an `AnticNumberField`
+  - `K` a finite extension of the rational function field over the
+     rationals
+  - `f` a polynomial over the rationals, an `AnticNumberField` or 
+     the univarite function field over the rationals (here explicit
+     permutations of the roots in a suitable splitting field are returned).
+     Futhermore, the the case of `f` over the rational function field, the
+     monodromy can be computed, ie. the automorphism group over the
+     complex numbers.
+
+Independently of the Galois group, subfields, that is intermediate fields
+between `K` and `k` can be computed as well.
+
+## Automorphism Group
+
+The automorphisms are computed using various specialised factoring
+algorithms: lifting the roots of the defining polynomial in the
+given field modulo suitable prime ideal powers and
+recovering the true roots from this information.
+
+The main information is included in the number field chaper, see
+
+  - [`automorphisms(::NumFieldMor)`](@ref)
+  - [`automorphism_group(::NumField)`](@ref)
+  - [`automorphism_group(::NumField, ::NumField)`](@ref)
+  - [`absolute_automorphism_group(::NumField)`](@ref)
+
+## Subfields
+
+The main information is included in the number field chaper, see
+
+  - [`subfields(K::SimpleNumField; degree::Int = -1)`](@ref)
+  - [`Hecke.principal_subfields(K::SimpleNumField)`](@ref)
+  - [`subfields(FF::Generic.FunctionField{fmpq})`](@ref)
+
+By setting `set_verbose_level(:Subfields, n::Int)` to 1 or 2
+information about the progress can be obtained.
+
+## Galois Group
+
+The computation of Galois groups follows Stauduhars algorithm with many
+improvements, see ... for an overview.
+
+The entrire computation can also be thought of finding a describtion of the
+splitting field of the polynomial. In fact, the information returned
+can be used to verify any algebraic identity between the roots, and
+find explicit subfields of the splitting field as well.
+
+Information about the progress is available via
+ 
+ - `set_verbose_level(:GaloisGroup, n::Int)`
+ - `set_verbose_level(:GaloisInvariants, n::Int)`
```

Comment:
Perhaps this would be better in the docstring for `galois_group`?

Reply:
it applies to all functions habing a galois in the name - and a few more. I don't know how to deal with this in a systematic, helpful way